utilities/storage.py

```diff
@@ -684,6 +688,31 @@ def write_file(vm, filename, content, stop_vm=True):
         vm.stop(wait=True)
 
 
+def write_file_via_ssh(vm: "VirtualMachineForTests", filename: str, content: str) -> None:
+    """
+    Write content to a file in VM using SSH connection.
+
+    Args:
+        vm: VirtualMachine instance with SSH connectivity
+        filename: Path to the file to write in the VM
+        content: Content to write to the file
+
+    Raises:
+        TimeoutExpiredError: If SSH connectivity cannot be established
+        SSHException: If SSH command execution fails
+    """
+
+    cmd = shlex.split(f"echo '{content}' > {filename} && sync")
+    run_ssh_commands(host=vm.ssh_exec, commands=cmd)
+
+
+def run_command_on_vm_and_check_output(vm: "VirtualMachineForTests", command: str, expected_result: str) -> None:
```

Comment:
this function eppears in https://github.com/RedHatQE/openshift-virtualization-tests/pull/3011- please do not add it to this PR
if needed, cherrypick once the above PR is merged

@jpeimer